templates/blog/topics/juju.html

```diff
@@ -0,0 +1,46 @@
+{% extends "templates/one-column.html" %} 
+
+{% block title %}Juju and JAAS{% endblock %}
+
+{% block meta_description %}Operate big software at scale on any cloud. All the latest news on Juju – the open source application modelling tool for public and private clouds.{% endblock %}
+
+{% block content %} 
+
+  <section class="p-strip--accent p-blog-topic__strip is-dark is-deep u-image-position" style="overflow: hidden; background-color: #333">
+    <div class="row header-strip u-equal-height">
+      <div class="col-6">
+        <div class="p-heading-icon">
+          <div class="p-heading-icon__header">
+            <img class="p-blog-heading__icon" src="{{ASSET_SERVER_URL}}67057b2c-product-page-juju-logo.svg" alt=""/>
+          </div>
+          <p class="p-heading--three">Operate big software at scale on any cloud</p>
+          <p>All the latest news on Juju – the open source application modelling tool for public and private clouds.</p>
+        </div>
+        <p>
+          <a class="p-link--external p-link--inverted" href="https://jujucharms.com">Learn more about Juju</a>
+        </p>
+      </div>
+      <div class="col-5 prefix-1">
+        <img
+          src="{{ASSET_SERVER_URL}}e8b86f15-Juju-Visual-Screen.png?w=576"
+          class="u-hidden--small u-image-position--bottom"
+          style="padding-top: 2.5rem; max-height: 100%;"
+          alt="" />
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip is-shallow" id="posts-list">
+    <div class="row u-equal-height u-clearfix">
```

Comment:
You should be able to replace this block with `{% include 'blog/posts.html' %}` (if you rebase)